### PR TITLE
fix: Bottom sheet title focus race condition

### PR DESCRIPTION
**Pull request by gorandalum**
Bottom sheet title were always focused, so when another ref was
added to the sheet content it was a race condition which ref was
focused by the screen reader.

Added a parameter to the BottomSheetContainer to specify whether
the title should be focused or not, and fixed up some of the bottom
sheets to use this parameter correctly.

### CODE REVIEW

**Review by adriawh — APPROVED** [review]
LGTM👍🏻

File: src/service-disruptions/ServiceDisruptionSheet.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const ServiceDisruptionSheet = forwardRef<View>(({}, focusRef) => {
         {hasValidServiceDisruptionUrl && (
           <>
             <Section style={style.serviceDisruptionText}>
-              <View ref={focusRef} accessible>
+              <View>
                 <ThemeText>{t(ServiceDisruptionsTexts.body)}</ThemeText>
               </View>
```
Comment (rosvik) [inline src/service-disruptions/ServiceDisruptionSheet.tsx:38]: This `<View>` looks redundant now
```suggestion
              <ThemeText>{t(ServiceDisruptionsTexts.body)}</ThemeText>
```

**Review by rosvik — APPROVED** [review]
LGTM 👍 Just one nitpick.

**Comment by tormoseng** [comment]
👍

**Pull request merged**